README.md

```diff
@@ -13,46 +13,43 @@ Supported AWS credentials:
   an [AWS secrets engine & sts-assumerole](https://www.vaultproject.io/docs/secrets/aws#sts-assumerole)
 * Fetching short-lived credentials from AWS via a OAuth2 authorization server
   and [OpenID Connect (OIDC)](https://openid.net/connect/)
+  * Additionally, you can fetch these credentials asynchronously
 
-For ready-to-use binaries have a look at releases. Additionally, we provide a _Docker image_ which can be used both in a
-test setup and as a sidecar in kubernetes.
+For ready-to-use binaries have a look at [Releases](https://github.com/idealo/aws-signing-proxy/releases).
 
-## 🎉 Version 2 Update 🎉
+Additionally, we provide a [Docker image](https://hub.docker.com/r/idealo/aws-signing-proxy) which can be used as a sidecar in Kubernetes.
+
+## 🎉 Version 2.0.0 Update 🎉
 
 * Version 2.0.0 comes 
   * with a built-in circuit breaker for requesting credentials from either OIDC or Vault 
   * with better error handling and panic recovery
   * with json logging enabled by default
 
-##### Vault Env Cred Provider
-
-In addition to the proxy you may also use `vault-env-cred-provider` as an
-[credential provider for AWS tooling](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html)
-.
+### Breaking Changes
 
-❗NOTE: the provided pre-built macOS binaries might fail with name resolution issues on your apple machine if you are
-using a (corporate) VPN. This will not occur on linux/windows/docker. If you are affected: either use the provided
-docker image or build the binaries on your machine.
+* Command line flags are now kebab-case to be POSIX style guide compliant
+* `Health Port` is now called `Mgmt Port` 
```

Comment:
Here is a part in the README missing, how to configure the MGMT Port and the Prometheus endpoint.

Reply:
Good point